src/devices/MemoryLCD/Extends/LSxxxB7DHxxExtends.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Drawing;
+using Iot.Device.MemoryLcd;
+
+namespace MemoryLcd.Extends
```

Comment:
Can you change the namespace to put this under Iot.Device.MemoryLcd as well please?